docs/data-management/post-session.md

```diff
@@ -374,11 +374,53 @@ ses-01
         └── sub-pilot_ses-01_task-rest_physio.tsv.gz
 ```
 
-
 ### Incorporate into version control with DataLad
 
 !!!info "Initiating the version-controled dataset"
 
-    Once at the beginning of the project, the datalad dataset will be created:
+    Once at the beginning of the project, the DataLad[1] dataset will be created:
 
     - [ ] Add stockage horus as an SSH remote.
+    - [ ] The preprocessing will be executed on an HPC. If datalad is not installed on the HPC, the most convenient user-based installation can be achieved by using Conda. 
+
+        !!!warning "Do not run the installation of Conda and DataLad in the login node"
+            I remind you that the login node is a shared resource meant for users to log in, submit jobs, and manage their workflows. It is recommended to use the login node only for tasks related to job submission, data management, and preparing job scripts. Indeed, running resource-intensive tasks or building containers on the login node can negatively impact the overall performance and responsiveness of the system for all users. Instead, open an interactive session in SLURM using the command below. You will need an interactive session to read the license and accept it.
+            ```
+            srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
+            ```
```

Comment:
Like above, this is specific to HPC, so if we end up running it locally, this would not have effect. Let me suggest the following changes:

- Make it a foldable admonition
- Downscale the priority to "tip"
- Separate new sentences into new lines for readability.

```suggestion
        ??? tip "Execution within High Performance Computing (HPC)"
        
            HPC systems typically recommend using their login nodes only for tasks related to job submission, data management, and preparing job scripts.
            Therefore, the execution of resource-intensive tasks such as *fMRIPrep* or building containers on login nodes can negatively impact the overall performance and responsiveness of the system for all users.
            Interactive sessions are a great alternative when available.
            For example, in the case of systems operating SLURM, the following command would open a new interactive session:
            ``` bash
            srun --nodes=1 --ntasks-per-node=1 --time=01:00:00 --pty bash -i
            ```
```

Reply:
@oesteban As you mentioned, the configuration commands were extracted from the DataLad Handbook.
However, I think having a condensed version of the relevant commands to execute our project is still valuable. Especially for us, in case we have to come back to these steps in one year, having them documented clearly would be of great help. Also, in the spirit of reproducibility, it is essential to detail the steps executed to obtain the results, so the SOPs seem like an excellent place to provide those details.

Overall, I like this solution because it puts less emphasis on those details, but they are not removed completely.